pineappl/src/boc.rs

```diff
@@ -571,12 +571,12 @@ impl Channel {
     /// use pineappl::boc::Channel;
     /// use pineappl::channel;
     ///
-    /// let entry = channel![103, 11, 10.0].translate(&|evol_id| match evol_id {
+    /// let entry = channel![10.0 * (103, 11)].translate(&|evol_id| match evol_id {
     ///     103 => vec![(2, 1.0), (-2, -1.0), (1, -1.0), (-1, 1.0)],
     ///     _ => vec![(evol_id, 1.0)],
     /// });
     ///
-    /// assert_eq!(entry, channel![2, 11, 10.0; -2, 11, -10.0; 1, 11, -10.0; -1, 11, 10.0]);
+    /// assert_eq!(entry, channel![10.0 * (2, 11) + -10.0 * (-2, 11) + -10.0 * (1, 11) + 10.0 * (-1, 11)]);
```

Comment:
I think you need to stress more that a literal `+` is needed, i.e. `10.0 * (2,11) - 10.0 * (-2, 11)` will not work (or support that as well), as people are used to "normal" math (where we precisely define that shortcut :upside_down_face: )

Reply:
I'm thinking of actually removing the macro altogether because it's only used in Rust, and there even only as a shortcut.

Reply:
well macros are in the first place short cuts, no?